dss/api/bundles/__init__.py

```diff
@@ -132,6 +133,41 @@ def get(
         response.headers['Link'] = link
         return response
 
+@dss_handler
+def enumerate(replica: str, prefix: str = 'bundles/', token: str = None,
+              per_page: int = PerPageBounds.per_page_max,
+              search_after: typing.Optional[str] = None):
+
+    if prefix and not prefix.startswith('bundles/'):
+        raise DSSException(requests.codes.bad_request, 'illegal_arguments',
+                           f'prefix: {prefix} must follow "bundle/" format')
+    api_domain_name = f'https://{os.environ.get("API_DOMAIN_NAME")}'
+    payload = dict(dss_api=api_domain_name, object='list', per_page=per_page,
+                   event_timestamp=datetime_to_version_format(datetime.datetime.utcnow()))  # type: typing.Any
+    kwargs = dict(replica=Replica[replica].name, prefix=prefix, per_page=per_page)
+    if search_after:
+        kwargs['search_after'] = search_after
+    if token:
+        kwargs['token'] = token
+
+    payload.update(list_available_uuids(**kwargs))  # type: ignore
+
+    if payload['page_count'] < per_page:
+        # enumeration is complete
+        payload.update(dict(has_more=False))
+        del payload['token']
+        del payload['search_after']
+        response = make_response(jsonify(payload), requests.codes.ok)
+    else:
+        next_url = UrlBuilder(request.url)
+        next_url.replace_query("search_after", payload['search_after'])
+        next_url.replace_query("token", payload['token'])
+        link = f"<{next_url}>; rel='next'"
+        payload.update(dict(has_more=True, token=payload['token'], link=f'{next_url}'))
+        response = make_response(jsonify(payload), requests.codes.partial)
+        response.headers['Link'] = link
+    return response
```

Comment:
Should return a status code too.  Something like `return jsonify(response_body), status_code`.